src/Board.js

```diff
@@ -12,36 +12,38 @@ const DropZone = props => {
         drop: () => { console.log("dropped on board"); props.onDrop(); }
     });
     console.log(props.cardsPlayed)
-    
+
     return (
         <div style={{textAlign: "center"}}>
-            Last card played:
+            Cards played:
             <div
                 ref={drop}
                 style={{
                     backgroundColor: 'lightgrey',
                     width: '100%',
                     height: '400px',
                     display: 'flex',
-                    flexDirection: 'column',
                     justifyContent: 'center',
                 }}
             >
                 {props.cardsPlayed.length > 0 ?
-                    <div 
-                        style={{
-                            color: 'black',
-                            fontSize: '24pt',
-                            width: '25%',
-                            height: '25%',
-                            textAlign: 'center',
-                            alignSelf: 'center',
-                        }}
-                    >
-                        {props.cardsPlayed[props.cardsPlayed.length-1]}
-                    </div>
-                    :
-                    null
+
+
+                        props.cardsPlayed.map((card) => (<div
+                            key={`card-${card.toString()}`} style={{
+                                color: 'black',
+                                background: 'aliceblue',
+                                fontSize: '24pt',
+                                width: '40px',
+                                padding: '4px 15px',
+                                marginRight: '10px',
+                                border: '1px solid #000',
+                                textAlign: 'center',
+                                alignSelf: 'center',
+                            }}
+                        >{card}</div>))
```

Comment:
i'm gonna merge anyways, i just thought i'd point out the indentation here is off-- try to stick to the existing indentation pattern next time.